subprojects/packagefiles/m4/checks/m4_test_runner.py

```diff
@@ -1,16 +1,15 @@
 import sys
 import subprocess
 import os
-import tempfile
 
 # We want to print line endings normally, but each line should be a b'' string
 def clean(x):
     lines = []
-    for l in x.splitlines():
-        if b'examples' in l:
-            lines.append(str(l.replace(b'\\',b'/')))
+    for line in x.splitlines():
+        if b'examples' in line:
+            lines.append(str(line.replace(b'\\',b'/')))
```

Comment:
This specific ruff "issue" is extremely opinionated and also wrong.

Reply:
I get that it's opinionated, and in this case I'd agree waht was there before was fine, I think the second loop is more readaable with a `line` vs `l` though, since it's  a long loop and things like `line[len('MY_REALLY_LONG_STRING):]` are generally more readable than `l[len('MY_REALLY_LONG_STRING):]`